frontend/src/components/PostsList.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import { useQuery } from '@tanstack/vue-query';
+import { UseTimeAgo } from '@vueuse/components';
+import AppLink from './AppLink.vue';
+
+const { data } = useQuery({
+  queryKey: ['posts'],
+  queryFn: async () => {
+    const request = await fetch('/api/posts?sort=recent');
+    const json = await request.json();
+    if (request.ok && json.success) {
+      console.log(json);
```

Comment:
This one seems redundant as well. You can look this information up in the network panel.